configs/default/TTRH-TRANSTECF411AIO.config

```diff
@@ -0,0 +1,138 @@
+# Betaflight / STM32F411 (S411) 4.2.4 Oct 20 2020 / 08:19:06 (fbcaf8c50) MSP API: 1.43
+
+board_name TRANSTECF411AIO
+manufacturer_id TTRH
+
+# resources
+resource BEEPER 1 B02
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B06
+resource MOTOR 4 B07
+resource MOTOR 5 B03
+resource MOTOR 6 B10
+resource PPM 1 A03
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource INVERTER 1 A14
+resource I2C_SCL 1 B08
```

Comment:
Seems to be a bit pointless to configure an I2C bus on an RTF model that does not use any I2C devices.